testing/src/main/java/com/linecorp/armeria/testing/server/ServerRule.java

```diff
@@ -156,10 +153,20 @@ public int httpPort() {
      * @throws IllegalStateException if the {@link Server} is not started or it did not open an HTTPS port
      */
     public int httpsPort() {
+        return port(HTTPS);
+    }
+
+    /**
+     * Returns the port number of the {@link Server} for the specified {@link SessionProtocol}.
+     *
+     * @throws IllegalStateException if the {@link Server} is not started or it did not open a port of the
+     *     specified protocol.
```

Comment:
indentation?

Reply:
This is the indentation checkstyle told me to use.

Reply:
Could look better if indentet at the same column with `if the..`

Reply:
Ah didn't realize the checkstyle was at least 4, not exactly 4 - fixed